keyboards/dubba175/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 tominabox1
```

Comment:
"tominabox1"?

Reply:
Apologies. I was looking at tominabox1's le_chiffre QMK branch for help understanding what changes I needed to make to dubba175 before submission.

Reply:
I have committed a change for it to read "drhigsby"